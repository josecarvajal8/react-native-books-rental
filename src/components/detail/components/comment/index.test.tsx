import React from 'react';
import renderer from 'react-test-renderer';
import Comment from '.';
const mockProps = {
    userName: 'testing',
    commentText: 'blablabla'
}
test('Render comment component correctly', () => {
    const commentCmp = renderer.create(<Comment {...mockProps} />).toJSON();
    expect(commentCmp).toMatchSnapshot();
});