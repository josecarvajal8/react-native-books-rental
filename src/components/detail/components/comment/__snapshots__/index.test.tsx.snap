// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render comment component correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "marginLeft": 20,
      "marginTop": 20,
      "maxHeight": 120,
      "width": "80%",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/img/default_profile.png",
      }
    }
    style={
      Object {
        "height": 60,
        "width": 60,
      }
    }
  />
  <View
    style={
      Object {
        "marginLeft": 20,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#000",
          "fontFamily": "Montserrat-SemiBold",
          "fontSize": 15,
        }
      }
    >
      testing
    </Text>
    <Text
      numberOfLines={4}
      style={
        Object {
          "fontFamily": "Montserrat-Regular",
          "fontSize": 15,
          "marginTop": 3,
          "width": "30%",
        }
      }
    >
      blablabla
    </Text>
  </View>
</View>
`;
